src/desktop/apps/auth/routes.coffee

```diff
@@ -23,4 +23,4 @@ sd = require('sharify').data
   res.render 'sign_up'
 
 @logIn = (req, res) ->
-  res.render 'log_in'
+  res.render 'log_in', redirect_to: (req.query.redirectTo || '/')
```

Comment:
As @yuki24 alluded to, I don't _think_ we'll need to add this here since the new auth app handles redirects.

Reply:
... I didn't have `sd.NEW_AUTH_MODAL` set locally. @yuki24

Reply:
Given that I see the new login page in an incognito window, I believe the new one has been rolled out to new users. Although I'd double-check with whoever has context.

Reply:
Yeah you'll want to use the new auth app, but looks like you guys figure that out.